### DISCUSSION TITLE
Since KC 21, no access to user-panel in Admin-Console: Cannot read properties of undefined

### DISCUSSION THREAD

**Discussion by DaBonse** [keycloak/keycloak#22769] [Q&A] [answered]
Hi,

Keycloak Version: 21.1.2 Docker

since Keycloak 21 we get no access to the user panel in Admin-Console in our realm. The master realm works fine.

We get the message:

`Cannot read properties of undefined (reading '0')`

Chrome debug toolbar says:

`[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952`

and

`TypeError: Cannot read properties of undefined (reading '0')
    at UsersSection.tsx:104:50
    at Array.filter (<anonymous>)
    at UsersSection.tsx:104:26
    at AdminClient.tsx:49:11`

Refreshing does not change anything. All other panels work as expected.

What can we do?

Thanks,

DaBonse

**Comment by jonkoops** [comment]
Could you report this as a bug instead? I think this is an issue that might need to be resolved in the Admin Console.

**Reply by DaBonse** [reply, marked as answer]
I have reported it as an issue:

https://github.com/keycloak/keycloak/issues/22803

**Comment by slominskir** [comment]
FYI this is still an issue in latest Keycloak (version 22.0.4).   The issue mentioned above says closed/fixed, but that doesn't appear to be true. 
![Screenshot](https://github.com/keycloak/keycloak/assets/2642119/371833c5-b9a0-4e7a-b96a-47c5fca013a5)

**Reply by jonkoops** [reply]
If you are still experiencing this issue please report it to the issue tracker so we can get it fixed.

**Reply by slominskir** [reply]
Created issue https://github.com/keycloak/keycloak/issues/23841
